src/sql/QueryEditor/FillValueSelect.tsx

```diff
@@ -43,7 +43,7 @@ export function FillValueSelect<TQuery extends DataQuery & Record<string, any>>(
               // Keep the fillMode.value in case FillValueOptions.Value mode is selected back
               fillMode: { ...props.query.fillMode, mode: value },
             });
-            props.onRunQuery();
```

Comment:
Would we ever have a situation here where someone would want to run the query on change but not on blur?

Reply:
Hmm, I don't think that situation occurs. Are you thinking we should remove this from the `onChange` handler completely? It would make sense to me to remove it, but I'm not familiar with all of the datasources that are relying on this behaviour from this component.

Reply:
Yeah I don't remember what I was getting at here haha sorry. I think there are weird nuances with onChange/onBlur and updating a query vs running a query that we sometimes get right from a ux perspective and sometimes don't, and I'd be wary of anything where we are changing the ux for all clients for this unless we've thought through why. But it seems like all the changes here are just making it optional which seems totally fine to me, since clients could already do this by just passing a no-op. So I'm not sure what I was thinking lol sorry! Looks good